code/src/main/java/com/google/sps/data/NegativeUserLocations.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+import java.util.ArrayList;
+
+@Entity
+public class NegativeUserLocations {
+
+  @Id String userId;
+  ArrayList<Location> locations;
+  long timeCreatedUnixTimeSeconds;
+
+  private NegativeUserLocations() {}
```

Comment:
I'm curious about having a private empty constructor? but if you have reasoning that's fine

Reply:
^^^throughout

Reply:
I made it private whenever there is a variable with getters and setters, however it seems like Grady wants all variables to be private and have getters and setters?

Reply:
yeah I think that's better practice

Reply:
Since it wasn't obvious to Cynthia or I that we have the empty constructor for Objectify, let's make it explicit.

Can you add a comment above or next to each empty constructor describing why we have it (nescessary for objectify?)

Reply:
done